docs/source/data-types/timeuuid.md

```diff
@@ -1,7 +1,7 @@
 # Timeuuid
 
 `Timeuuid` is represented as `value::CqlTimeuuid`.
-`value::CqlTimeuuid` is a wrapper  for `uuid::Uuid` with custom ordering logic
+`value::CqlTimeuuid` is a wrapper for `uuid::Uuid` (using the `v1` feature) with custom ordering logic
```

Comment:
The _using the `v1` feature_ part should only be said be needed when _constructing_ `CqlTimeuuid`; using them does not require the feature. Please rewrite this accordingly.